tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -426,23 +426,11 @@ const reducer = createReducer(
   }),
   on(globalSettingsLoaded, (state, {partialSettings}) => {
     const metricsSettings: Partial<MetricsSettings> = {};
-    if (partialSettings.tooltipSortString) {
-      switch (partialSettings.tooltipSortString) {
-        case TooltipSort.DEFAULT:
-        case TooltipSort.ALPHABETICAL:
-          metricsSettings.tooltipSort = TooltipSort.ALPHABETICAL;
-          break;
-        case TooltipSort.ASCENDING:
-          metricsSettings.tooltipSort = TooltipSort.ASCENDING;
-          break;
-        case TooltipSort.DESCENDING:
-          metricsSettings.tooltipSort = TooltipSort.DESCENDING;
-          break;
-        case TooltipSort.NEAREST:
-          metricsSettings.tooltipSort = TooltipSort.NEAREST;
-          break;
-        default:
-      }
+    if (
+      partialSettings.tooltipSortString &&
+      Object.values(TooltipSort).includes(partialSettings.tooltipSortString)
+    ) {
+      metricsSettings.tooltipSort = partialSettings.tooltipSortString;
```

Comment:
Is it ok that partialSettings.tooltipSortString === TooltipSort.DEFAULT will now set metricsSettings.tooltipSort to TooltipSort.DEFAULT? This is a slight change in behavior of the reducer and its not clear to me whether that's a problem.

If it turns out the change in behavior is NOT OK then can we add a test for that case?

Reply:
This should be okay because when the sorting is actually done in `tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ts` they use the same logic.